remove shard attribute, related functionality and examples

yhy0217 opened the pull request:
- [x] Closes #91 
- [x] Update the extension image of the French version documentation

prushforth commented:
Addresses https://github.com/Maps4HTML/MapML/issues/233

prushforth reviewed (approved):
I restored the \<input hidden> attribute, minus a mention of shard, because I think hidden inputs are legit, still.  I am sorry to lose map-datalist, in a way, but if we ever use it the page is not deleted.  I will update MapML to remove map-datalist as well.

I don't see why the image file here is deleted?  Pls confirm.

yhy0217 commented:
> I restored the \<input hidden> attribute, minus a mention of shard, because I think hidden inputs are legit, still.  I am sorry to lose map-datalist, in a way, but if we ever use it the page is not deleted.  I will update MapML to remove map-datalist as well.

Sounds good! :)

> I don't see why the image file here is deleted?  Pls confirm.

The image file demonstrates the UI of the mapml-extension, and since I changed the UI in my previous PR (to add user settings for copy operations), the image should be updated too.
I believed I have updated the image in assets\img folder when I worked on my prev PR; But seems that the english and french versions now no longer share the same assets folder, so I just updated it again for the French version documentation.
